generation/anomaly_detection/2d_classifierfree_guidance_anomalydetection_tutorial.ipynb

```diff
@@ -0,0 +1,1033 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #91.    )  # np.linspace(1, n_iterations, len(val_iter_loss_list))
</pre><p>Please remove this comment and any other bits of commented code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>